app/policies/language_policy.rb

```diff
@@ -1,11 +1,23 @@
 class LanguagePolicy < ApplicationPolicy
-  MANAGE_LANGUAGES = %w[superadmin translation].freeze
+  LANGUAGE_EDIT_ACCESS = %w[superadmin translation support policy_and_abuse].freeze
+  LANGUAGE_CREATE_ACCESS = %w[superadmin translation].freeze
 
   def new?
-    user_has_roles?(MANAGE_LANGUAGES)
+    user_has_roles?(LANGUAGE_CREATE_ACCESS)
+  end
+
+  def edit?
+    user_has_roles?(LANGUAGE_EDIT_ACCESS)
+  end
+
+  def can_edit_abuse_support_available?
+    user_has_roles?(%w[superadmin translation policy_and_abuse])
+  end
+
+  def can_edit_other_fields?
```

Comment:
I think this name makes more sense when you don't have the context of the other check:
```suggestion
  def can_edit_non_abuse_fields?
```